config/config.go

```diff
@@ -43,47 +38,25 @@ type Config struct {
 		Level       string
 	}
 
-	Core struct {
-		Image string
-		Name  string
-		Path  string
-
-		Database struct {
-			ServiceRelativePath   string
-			ExecutionRelativePath string
-		}
+	Database struct {
+		ServiceRelativePath   string
+		ExecutionRelativePath string
 	}
 
 	Service ServiceConfigGroup
-
-	Docker struct {
-		Socket string
-		Core   struct {
-			Path string
-		}
-	}
 }
 
 // New creates a new config with default values.
 func New() (*Config, error) {
-	home, err := homedir.Dir()
-	if err != nil {
-		return nil, err
-	}
-
 	var c Config
 	c.Server.Address = ":50052"
-	c.Client.Address = "localhost:50052"
 	c.Log.Format = "text"
 	c.Log.Level = "info"
 	c.Log.ForceColors = false
-	c.Core.Image = "mesg/engine:" + strings.Split(version.Version, " ")[0]
-	c.Core.Name = "engine"
-	c.Core.Path = filepath.Join(home, ".mesg")
-	c.Core.Database.ServiceRelativePath = filepath.Join("database", "services", serviceDBVersion)
-	c.Core.Database.ExecutionRelativePath = filepath.Join("database", "executions", executionDBVersion)
-	c.Docker.Core.Path = "/mesg"
-	c.Docker.Socket = "/var/run/docker.sock"
+	c.Name = "engine"
+	c.Path = filepath.Join("mesg")
```

Comment:
```suggestion
	c.Path = "mesg"
```

or home is missing? also if mesg will be created in engine package, it is worth to add `mesg` to .gitignore

Reply:
It was a mistake, I put back path to `/path`.
This path is only used inside the docker container.